backend/src/test/java/com/odde/doughnut/controllers/RestQuizQuestionAndAnswerControllerTests.java

```diff
@@ -509,7 +509,8 @@ void setUp() {
 
     @Test
     void delete() throws UnexpectedNoAccessRightException {
-      QuizQuestionAndAnswer questionAndAnswer = makeMe.aQuestion().please();
+      QuizQuestionAndAnswer questionAndAnswer =
+          makeMe.aQuestion().approvedSpellingQuestionOf(subjectNote).please();
```

Comment:
Can this problem be avoided if we just remove the note id from the API parameter?

People like USB C over USB A because with USB A there's > 50% change of making a mistake.

I guess what matters more is if the user have access to the note.

Reply:
Oh, I think I kinda get your point after I check my API endpoint. I didn't check it first.

<img width="449" alt="Screenshot 2567-07-13 at 20 26 27" src="https://github.com/user-attachments/assets/7acb5475-cbbe-452f-bf6c-1ce447d4c20a">
 
Yes, it makes sense to remove the note ID from the API parameter. Thanks a lot!